environments/india/public.yml

```diff
@@ -44,6 +44,9 @@ kafka_log_dir: "{{ encrypted_root }}/kafka"
 
 rabbitmq_version: 3.8.5-1
 
+#RabbitMQ Broker URL
+BROKER_URL: 'amqp://{{ AMQP_USER }}:{{ AMQP_PASSWORD }}@{{ AMQP_HOST }}:5672/{{ AMQP_NAME }};amqp://{{ AMQP_USER }}:{{ AMQP_PASSWORD }}@{{ groups.rabbitmq.1 }}:5672/{{ AMQP_NAME }}'
+
```

Comment:
Naively my first thought was you could probably also do this by just switching `kafka1` and `rabbit2` under `[rabbitmq:children]` in the inventory above. Does something else rely on that order that you need to stay the same?

Reply:
Yes, Danny. The sequence should be the same. Because kafka1 is right now acting as Master node. Please let me know if we are missing anything. thanks